openapi.json

```diff
@@ -12,295 +12,283 @@
     ],
     "paths": {
         "/applications/{applicationId}": {
-            "$ref": "./schemas/application_paths.json#/application"
+            "$ref": "./schemas/application/application_paths.json#/application"
         },
         "/applications/{applicationId}/cancel": {
-            "$ref": "./schemas/application_paths.json#/cancel_application"
+            "$ref": "./schemas/application/application_paths.json#/cancel_application"
         },
         "/applications": {
-            "$ref": "./schemas/application_paths.json#/applications"
+            "$ref": "./schemas/application/application_paths.json#/applications"
         },
         "/application-forms": {
-            "$ref": "./schemas/application_paths.json#/application_forms"
+            "$ref": "./schemas/application/application_paths.json#/application_forms"
         },
         "/applications/{applicationId}/documents/{documentId}": {
-            "$ref": "./schemas/application_paths.json#/upload_pdf_file"
-        },
-        "/applications/{applicationId}/documents/{documentId}/": {
-            "$ref": "./schemas/application_paths.json#/upload_png_file"
-        },
-        "/applications/{applicationId}/documents/{documentId}/?": {
-            "$ref": "./schemas/application_paths.json#/upload_jpg_file"
+            "$ref": "./schemas/application/application_paths.json#/upload_application_document_file"
         },
         "/applications/{applicationId}/documents/{documentId}/back": {
-            "$ref": "./schemas/application_paths.json#/upload_pdf_file_back"
-        },
-        "/applications/{applicationId}/documents/{documentId}/back/": {
-            "$ref": "./schemas/application_paths.json#/upload_png_file_back"
-        },
-        "/applications/{applicationId}/documents/{documentId}/back/?": {
-            "$ref": "./schemas/application_paths.json#/upload_jpg_file_back"
+            "$ref": "./schemas/application/application_paths.json#/upload_application_document_file_back"

```

Comment:
This specifically was a kludge, open api doens't support different definitions for the same path, so I've updated the schemas properly and removed improper path definitions